modules/scalatest/src/weaver/scalatestcompat/Matchers.scala

```diff
@@ -0,0 +1,27 @@
+package weaver.scalatestcompat
+
+import cats.Monoid
+import weaver.{ EffectSuite, Expectations, SourceLocation }
+import weaver.Expectations.Helpers._
+import org.scalatest.Assertion
+import scala.util.{ Failure, Success, Try }
+
+trait Matchers[F[_]] {
+  self: EffectSuite[F] =>
+
+  implicit def toExpectations(
+      a: Assertion
+  )(
+      implicit pos: SourceLocation
+  ): Expectations =
+    Try(a) match {
+      case Success(_) => Monoid[Expectations].empty
+      case Failure(t) => failure(t.getMessage)
```

Comment:
scalatest throws a [TestFailedException](https://www.scalatest.org/scaladoc/3.2.5/org/scalatest/exceptions/TestFailedException.html) to indicate that a test failed.  Here we fail on any exception. Comments?